app/Admin/src/Http/Controllers/Auth/AuthController.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace App\Admin\Http\Controllers\Auth;
+
+use App\Admin\Http\Requests\Auth\LoginRequest;
+use App\Admin\Http\Requests\Auth\LogoutRequest;
+use App\Platform\Enums\RoleName;
+use Illuminate\Http\Response;
+use Illuminate\Support\Facades\Auth;
+use Illuminate\Validation\ValidationException;
+use Knuckles\Scribe\Attributes\Authenticated;
+use Knuckles\Scribe\Attributes\BodyParam;
+use Knuckles\Scribe\Attributes\Endpoint;
+
+class AuthController
+{
+    /**
+     * @throws ValidationException
+     */
+    #[Endpoint('Admin/Login', <<<'DESC'
+    This endpoint enable users with admin role to sign in
+ DESC)]
+    #[BodyParam('email', 'password', required: true)]
+    public function login(LoginRequest $request)
+    {
+        $credentials = $request->only('email', 'password');
+
+        if (Auth::attempt($credentials)) {
```

Comment:
You should find the admin by email first before you attempt to log in. 
Currently, you are logging in first and then you are trying to check if the admin has the role.

Reply:
@davorminchorov instead of checking admin by email in controller, can this be done in the request class with validation rule `'ends_with:learnhub.mk'`?

Reply:
Yeah, that works as well.